content/hi/docs/reference/glossary/ingress.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Ingress
```

Comment:
This needs to be in Hindi

Reply:
You can use `aka:` to also provide the original name.